src/pasta/mod.rs

```diff
@@ -38,6 +38,9 @@ const ENDO_PARAMS_EP: EndoParameters = EndoParameters {
 endo!(Eq, Fp, ENDO_PARAMS_EQ);
 endo!(Ep, Fq, ENDO_PARAMS_EP);
 
+// prime_field_legendre!(Fp);
+// prime_field_legendre!(Fq);
```

Comment:
Calling this macro here results in conflicting names for the values generated in the `lazy_static!`. I have tried the following:
``` rust
static ref [< $field _LE_AS_BIGUINT >]: BigUint = BigUint::from_bytes_le((-<$field as ff::Field>::ONE).to_repr().as_ref())/2usize ;
static ref [< $field _LEGENDRE_EXP >]: Vec<u64> = [< $field _LE_AS_BIGUINT >].to_u64_digits();
```
inside a `paste!`.
But I get some annoying warnings regarding non upper case globals that I haven't managed to remove